generation/day_ahead.py

```diff
@@ -114,6 +114,12 @@ def import_day_ahead_generation(
     abort_if_data_empty(scheduled_generation)
     log.debug("Overall aggregated generation: \n%s" % scheduled_generation)
 
+    log.debug("Up-sampling overall aggregated generation ...")
```

Comment:
We only need to log that we're about to do this, or have done it, when we actually have to do it (inside the if-block)